lefthook.yml

```
@@ -0,0 +1,66 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format sidebars
+      glob: "files/sidebars/*.yaml"
+      run: yarn content fmt-sidebars
+      stage_fixed: true
+
+    - name: Fix redirects
+      glob: "files/en-us/_redirects.txt"
+      run: yarn content fix-redirects en-US && yarn content validate-redirects en-US
+      stage_fixed: true
+
+    - name: Format non-md
```

Nit: for consistency

```suggestion
    - name: Format non-markdown
```